src/live.asp.net/ViewModels/HomeViewModel.cs

```diff
@@ -9,6 +9,8 @@ namespace live.asp.net.ViewModels
 {
     public class HomeViewModel
     {
+        private static string _dateTimeFormat = "yyyyMMddTHHmmssZ";
```

Comment:
Agree it should be `readonly` but why `public`? The value isn't used, nor intended to be used, outside of the view model.

I followed your lead from [iCalendarOutputFormatter.cs](https://github.com/aspnet/live.asp.net/blob/dev/src/live.asp.net/Formatters/iCalendarOutputFormatter.cs). Should I refactor? To say DateTimeExtensions.ToIso8601()?